### PR TITLE
fix: placement should be center when target is null 

### LINKED ISSUE #43673: Tour 组件 target={null} 时弹窗不垂直居中
### Reproduction link

[https://ant.design/components/tour-cn](https://ant.design/components/tour-cn)

### Steps to reproduce

点击「位置」官方用例的按钮就能直接复现。
image

### What is expected?

引导卡片垂直居中

### What is actually happening?

在 antd 版本 ~5.2 是正常的，从 ~5.3 开始一直异常。当引导内容较多时（引导卡片趋近正方形），视觉问题突出的就更明显。

| Environment | Info |
| --- | --- |
| antd | 5.7.1 |
| React | 18.1.0 |
| System | MacOS 13.4.1 (22F82) |
| Browser | Chrome 114.0.5735.198 |

---
在 mac 上（1280*800）卡片居上（卡片的下边沿出现在屏幕中间），但在显示器上（1920*1080）卡片居下（卡片的上边沿出现在屏幕中间）

### PR DESCRIPTION
**Pull request by linxianxi**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/43673
close #43742

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    -       |
| 🇨🇳 Chinese |    -       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 169d164</samp>

Updated the `Tour` and `TourStep` components and their documentation to support the `center` option for the `placement` property. This option allows the tour guide to be centered on the screen or on the target element, improving the flexibility and user experience of the component.

### <samp>🤖 Generated by Copilot at 169d164</samp>

*  Add `center` option to `placement` property of `Tour` and `TourStep` components, which allows the guide to be centered on the screen when there is no target element or when each step specifies it ([link](https://github.com/ant-design/ant-design/pull/43694/files?diff=unified&w=0#diff-c90ae517d30ca76723f551bc647174d1abc7fc62f6e421be6f5d8d0ac28e92baR14), [link](https://github.com/ant-design/ant-design/pull/43694/files?diff=unified&w=0#diff-1ad8b229c044ca1a06a7bbed19bc03af75d71e159a8b70caf2a52aa2e328b38aL34-R34), [link](https://github.com/ant-design/ant-design/pull/43694/files?diff=unified&w=0#diff-1ad8b229c044ca1a06a7bbed19bc03af75d71e159a8b70caf2a52aa2e328b38aL54-R54), [link](https://github.com/ant-design/ant-design/pull/43694/files?diff=unified&w=0#diff-8eeca01fdc999bcb32068854b6930fb0df7f5374284c2d03e3a0b7fb6d553d8cL35-R35), [link](https://github.com/ant-design/ant-design/pull/43694/files?diff=unified&w=0#diff-8eeca01fdc999bcb32068854b6930fb0df7f5374284c2d03e3a0b7fb6d553d8cL56-R56))
* Update `placement.md` and `placement.tsx` files in `components/tour/demo` folder to demonstrate the new feature and match the documentation ([link](https://github.com/ant-design/ant-design/pull/43694/files?diff=unified&w=0#diff-167f8197ac1fc6418e391d047a710c2f2636fccdee5c41340d3d296511555f7fL3-R7), [link](https://github.com/ant-design/ant-design/pull/43694/files?diff=unified&w=0#diff-c90ae517d30ca76723f551bc647174d1abc7fc62f6e421be6f5d8d0ac28e92baR14))

### CODE REVIEW

File: components/tour/demo/placement.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## zh-CN
 
-改变引导相对于目标的位置，共有 12 种位置可供选择。当 `target={null}` 时引导将会展示在正中央。
+改变引导相对于目标的位置，共有 12 种位置可供选择。当 `target={null}` 和 `placement='center'` 时引导将会展示在正中央。
```
Comment (afc163) [inline components/tour/demo/placement.md:3]: 不对，target 为 null 时就应该在中间。不需要额外单独配置 placement

Comment (MuxinFeng) [reply components/tour/demo/placement.md:3]: 在 rc-tour 里，当 target 为 null 时，如果 placement 未设置，默认为 center 如何

Comment (afc163) [reply components/tour/demo/placement.md:3]: +1

**Comment by afc163** [comment]
> 在 rc-tour 里，当 target 为 null 时，如果 placement 未设置，默认为 center 如何

**Review by afc163 — CHANGES_REQUESTED** [review]
> 在 rc-tour 里，当 target 为 null 时，如果 placement 未设置，默认为 center 如何

**Comment by linxianxi** [comment]
> > 在 rc-tour 里，当 target 为 null 时，如果 placement 未设置，默认为 center 如何

上周改了，等豆酱 rc-tour 那边合

**Review by afc163 — APPROVED** [review]

**Pull request merged**